frameworks/hdfs/tests/test_sanity.py

```diff
@@ -148,10 +148,21 @@ def test_kill_data_node():
 @pytest.mark.sanity
 @pytest.mark.recovery
 def test_kill_scheduler():
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+    scheduler_task_name = sdk_utils.get_task_id_service_name(foldered_name)
+    scheduler_id = sdk_tasks.get_task_ids(foldered_name, scheduler_task_name)
+    journal_ids = sdk_tasks.get_task_ids(foldered_name, "journal")
+    name_ids = sdk_tasks.get_task_ids(foldered_name, "name")
+    data_ids = sdk_tasks.get_task_ids(foldered_name, "data")
+
     sdk_cmd.kill_task_with_pattern(
         "hdfs.scheduler.Main", shakedown.get_service_ips("marathon").pop()
     )
-    config.check_healthy(service_name=sdk_utils.get_foldered_name(config.SERVICE_NAME))
+    config.check_healthy(service_name=foldered_name)
```

Comment:
We can't call `check` healthy here. There is not scheduler running at this point. (Since you're out, I'll push a change)